webview/src/plots/components/ZoomedInPlot.tsx

```diff
@@ -60,13 +62,26 @@ export const ZoomedInPlot: React.FC<ZoomedInPlotProps> = ({
     }
   }, [])
 
-  const onNewView = () => {
+  const onNewView = (view: View) => {
     const actions: HTMLDivElement | null | undefined =
       zoomedInPlotRef.current?.querySelector('.vega-actions')
     if (!actions) {
       return
     }
 
+    appendActionToVega('SVG', actions, () => {
+      void view.toSVG().then(svg => {
```

Comment:
[Q] @julieg18 do you remember if there is a way to close the menu on click?

Reply:
Looks like `;(vegaActions.parentNode as HTMLElement).removeAttribute('open')` works

Reply:
> ;(vegaActions.parentNode as HTMLElement).removeAttribute('open') works

Yes, we adjust the same attribute when we're opening the menu automatically for the `ZoomedInPlot`.